std/uni.d

```diff
@@ -9108,7 +9108,15 @@ bool isMark(dchar c)
 @safe pure nothrow @nogc
 bool isNumber(dchar c)
 {
-    return numberTrie[c];
+    // optimization
+    if (c < 0xAA)
```

Comment:
I wonder if there would be a perf difference between this and:

```
if (c >= 48 && c <= 57)
   return true;
else
  return numberTrie[c];
```

Otherwise, LGTM.